### PR TITLE
week-05

### PR DESCRIPTION
**Pull request by Vikashrock45**

### CODE REVIEW

File: week-04/4.2/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            // const innerTitle = document.createElement('div');
+            // const innerDescription = document.createElement('div');
+            // const innerButton = document.createElement('button');
+            // innerTitle.innerHTML = title
+            // innerDescription.innerHTML = description
+            // innerButton.innerHTML = 'Mark as Done'
+            // outerDiv.appendChild(innerTitle);
+            // outerDiv.appendChild(innerDescription);
+            // outerDiv.appendChild(innerButton);
+            // document.getElementById('container').appendChild(outerDiv);
```
Comment (vikashrock4545) [inline week-04/4.2/index.html:52]: Why so much comment

File: week-04/4.2/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        }
+        // updateDomAcToState([{
+        //     title: "go to gym",
+        //     description: "Go to gym from 6-9",
+        //     id: 1
+        // },{
+        //     title: "go to gym",
+        //     description: "Go to gym from 10-12",
+        //     id: 2
+        // }])
```
Comment (vikashrock4545) [inline week-04/4.2/index.html:70]: again

File: week-04/4.2/index.html (hunk 1 above)
Comment (Vikashrock45) [reply week-04/4.2/index.html:52]: Done.

File: week-04/4.2/index.html (hunk 2 above)
Comment (Vikashrock45) [reply week-04/4.2/index.html:70]: Done

**Pull request merged**